src/TensorFlowNET.Core/Keras/Engine/ICallback.cs

```diff
@@ -15,5 +15,5 @@ public interface ICallback
     void on_predict_end();
     void on_test_begin();
     void on_test_batch_begin(long step);
-    void on_test_batch_end(long end_step, IEnumerable<(string, Tensor)> logs);
+    void on_test_batch_end(long end_step, Dictionary<string, float> logs);
```

Comment:
Since I rectify the definition of logs in Model.Evaluate, so here and somewhere need to change either.